external/libdxfrw/drw_entities.cpp

```diff
@@ -220,72 +220,99 @@ bool DRW_Entity::parseDxfGroups( int code, dxfReader *reader )
 bool DRW_Entity::parseDwg( DRW::Version version, dwgBuffer *buf, dwgBuffer *strBuf, duint32 bs )
 {
   objSize = 0;
+#ifdef DWGDEBUG
```

Comment:
doe it need documentation?

Reply:
I don't think so. It's only targetted as developers who will find their way to enable it. If it was worth it, it should go to a CMake option, but IMHO the only effect of this would be a worse signal to noise ratio in those options.

Reply:
well would be useful empower the user to find himself bugs or at least generate more detailed logs if necessary to suport bugfix... more or less in the same way it's documented in https://qgis.org/api/classQgsLogger.html#details nothing more than this.

Reply:
I've added a comment to the top of the page. It's a compile time flag, so there's not much point in exposing this to end users.